frontends/election-manager/src/screens/print_test_deck_screen.tsx

```diff
@@ -117,72 +163,97 @@ export function PrintTestDeckScreen(): JSX.Element {
       const printers = await window.kiosk.getPrinterInfo();
       if (printers.some((p) => p.connected)) {
         setPrecinctIds(generatePrecinctIds(precinctId));
-        setPrecinctIndex(0);
+        setPrintIndex({ precinctIndex: 0, component: 'tally' });
       } else {
         // eslint-disable-next-line no-alert
         window.alert('please connect the printer.');
         await logger.log(LogEventId.TestDeckPrinted, currentUserType, {
           disposition: 'failure',
-          message: `Failed to print test deck: no printer connected.`,
+          message: `Failed to print L&A Package: no printer connected.`,
           result: 'User shown error message, asked to try again.',
           error: 'No printer connected.',
         });
       }
     } else {
       setPrecinctIds(generatePrecinctIds(precinctId));
-      setPrecinctIndex(0);
+      setPrintIndex({ precinctIndex: 0, component: 'tally' });
     }
   }
 
-  const onAllRendered = useCallback(
+  const onPrecinctTallyReportRendered = useCallback(
+    async (numPages) => {
+      if (!printIndex) {
+        return;
+      }
+
+      await printer.print({ sides: 'one-sided' });
+      await logger.log(LogEventId.TestDeckTallyReportPrinted, currentUserType, {
+        disposition: 'success',
+        message: `Test deck tally report printed as part of L&A package for precinct id: ${
+          precinctIds[printIndex.precinctIndex]
+        }`,
+        precinctId: precinctIds[printIndex.precinctIndex],
+      });
+
+      await makeCancelable(sleep(numPages * 3000));
+      setPrintIndex({
+        precinctIndex: printIndex.precinctIndex,
+        component: 'hmpb',
+      });
```

Comment:
Order-wise, I think we actually want the tally report to come after the HMPBs, the thinking being that you scan the test deck ballots and then verify that the tallies match the expected tallies

Reply:
I put it last based on [Ginny's comment in the design doc](https://docs.google.com/document/d/1M7n9fR7VE-qmcEqpW_lFVW3BCFZBwkbhppv9Q2_qJ2Y/edit?disco=AAAAY2nnZfQ). I like it at the front because, since we're not doing spacers, it serves as a signal that this is the next L&A package when printing all of them. But I don't have a strong opinion, and definitely see your point. 

I think the critical piece of ordering is that the VxMark test deck comes before the VxAdmin test deck, since the VxAdmin is padded with the "extra" ballots at the end and it's good to have the two proper test decks be consecutive.

Reply:
Ah nice! I'd missed that comment and like Ginny's reasoning for putting the tally report first